### ISSUE TITLE
PHP notice: Undefined variable: classKeyIcon

### ISSUE THREAD

**Issue by Ibochkarev** [modxcms/revolution#15501]
### Summary
Log error:

```
[2021-03-06 13:51:52] (WARN @ /Users/bochkarev/Sites/bughunt/core/src/Revolution/Processors/Resource/GetNodes.php : 494) PHP notice: Undefined variable: classKeyIcon
```

### Step to reproduce
Set system setting `log_level` to 4

Go to resource editing, go to the Settings tab and toggle the Container checkbox, then click the save button.

### Observed behavior
```
[2021-03-06 13:51:52] (WARN @ /Users/bochkarev/Sites/bughunt/core/src/Revolution/Processors/Resource/GetNodes.php : 494) PHP notice: Undefined variable: classKeyIcon
```

### Expected behavior
There should be no errors in the log

### Related issue(s)/PR(s)
NA

### Environment
MODX Revolution 3.0.0-alpha3 (git), PHP Version 7.4.12

**Comment by Mark-H** [comment]
Related to https://github.com/modxcms/revolution/issues/14857 - there's various warnings/issues in that specific processor.

**Comment by Ibochkarev** [comment]
https://user-images.githubusercontent.com/2138260/132119658-ce4746c2-3247-4eaf-97cb-dd0e18b73799.mp4

**Comment by theboxer** [comment]
This should be resolved as #15834 got merged
